src/openrct2/scripting/bindings/entity/ScGuest.cpp

```diff
@@ -12,9 +12,138 @@
+        { "hat_much", PeepThoughtType::HatMuch },
+        { "toffee_apple_much", PeepThoughtType::ToffeeAppleMuch },
+        { "tshirt_much", PeepThoughtType::TshirtMuch },
+        { "doughnut_much", PeepThoughtType::DoughnutMuch },
+        { "coffee_much", PeepThoughtType::CoffeeMuch },
+        { "chicken_much", PeepThoughtType::ChickenMuch },
+        { "lemonade_much", PeepThoughtType::LemonadeMuch },
+        { "photo2", PeepThoughtType::Photo2 },
+        { "photo3", PeepThoughtType::Photo3 },
+        { "photo4", PeepThoughtType::Photo4 },
+        { "pretzel", PeepThoughtType::Pretzel },
+        { "hot_chocolate", PeepThoughtType::HotChocolate },
+        { "iced_tea", PeepThoughtType::IcedTea },
+        { "funnel_cake", PeepThoughtType::FunnelCake },
+        { "sunglasses", PeepThoughtType::Sunglasses },
+        { "beef_noodles", PeepThoughtType::BeefNoodles },
+        { "fried_rice_noodles", PeepThoughtType::FriedRiceNoodles },
+        { "wonton_soup", PeepThoughtType::WontonSoup },
+        { "meatball_soup", PeepThoughtType::MeatballSoup },
+        { "fruit_juice", PeepThoughtType::FruitJuice },
+        { "soybean_milk", PeepThoughtType::SoybeanMilk },
+        { "sujongkwa", PeepThoughtType::Sujongkwa },
+        { "sub_sandwich", PeepThoughtType::SubSandwich },
+        { "cookie", PeepThoughtType::Cookie },
+        { "roast_sausage", PeepThoughtType::RoastSausage },
+        { "photo2_much", PeepThoughtType::Photo2Much },
+        { "photo3_much", PeepThoughtType::Photo3Much },
+        { "photo4_much", PeepThoughtType::Photo4Much },
+        { "pretzel_much", PeepThoughtType::PretzelMuch },
+        { "hot_chocolate_much", PeepThoughtType::HotChocolateMuch },
+        { "iced_tea_much", PeepThoughtType::IcedTeaMuch },
+        { "funnel_cake_much", PeepThoughtType::FunnelCakeMuch },
+        { "sunglasses_much", PeepThoughtType::SunglassesMuch },
+        { "beef_noodles_much", PeepThoughtType::BeefNoodlesMuch },
+        { "fried_rice_noodles_much", PeepThoughtType::FriedRiceNoodlesMuch },
+        { "wonton_soup_much", PeepThoughtType::WontonSoupMuch },
+        { "meatball_soup_much", PeepThoughtType::MeatballSoupMuch },
+        { "fruit_juice_much", PeepThoughtType::FruitJuiceMuch },
+        { "soybean_milk_much", PeepThoughtType::SoybeanMilkMuch },
+        { "sujongkwa_much", PeepThoughtType::SujongkwaMuch },
+        { "sub_sandwich_much", PeepThoughtType::SubSandwichMuch },
+        { "cookie_much", PeepThoughtType::CookieMuch },
+        { "roast_sausage_much", PeepThoughtType::RoastSausageMuch },
+        { "help", PeepThoughtType::Help },
+        { "running_out", PeepThoughtType::RunningOut },
+        { "new_ride", PeepThoughtType::NewRide },
+        { "nice_ride_deprecated", PeepThoughtType::NiceRideDeprecated },
+        { "excited_deprecated", PeepThoughtType::ExcitedDeprecated },
+        { "here_we_are", PeepThoughtType::HereWeAre },
+    });
```

Comment:
Maybe it would be useful to do something like:
```c++
static_assert(ThoughtTypeMap.size() == static_cast<uint8_t>(PeepThoughtType::None), "Mismatch of thought mapping between code and plugin API");
```

Or an assertion, if we can't do a `static` one.

Reply:
The values in `PeepThoughtType` are not strictly consecutive, including `PeepThoughtType::None` which is 255

Reply:
Ah, true